README.md

```diff
@@ -32,6 +32,17 @@ A Dockerfile for Sonatype Nexus IQ Server, based on [Red Hat Universal Base Imag
 * [Project License](#project-license)
 
 ## Migration
+
+### Upgrading from Version 100 or Earlier to Version 101 or Later
+Version 1.101.0 of the Docker image changed the base image from Red Hat UBI (Universal Base Image) to include
+OpenJDK 1.8. As a result, the UID of the `nexus` user has changed. If you use this image with a persistent data volume
+you will need to run the following for your server to start up successfully:
+```
+docker run -it -u=0 -v sonatype-work:/sonatype-work sonatype/nexus-iq-server:1.101.0 chown -R nexus:nexus /sonatype-work
+```
+This will start up a 1.101.0 IQ server container with root as the user, allowing it to chown the sonatype-work directory
+and its files to the correct nexus user.
```

Comment:
NOTE: I see two other places that say `nexus` user, not sure if this is intentionally different or oversight.

Reply:
> NOTE: I see two other places that say `nexus` user, not sure if this is intentionally different or oversight.

Fixed in https://github.com/sonatype/docker-nexus-iq-server/pull/52/commits/9162762c1fa2f30fa9aadbdfd0fedf946028cdc2